projects/vdk-plugins/quickstart-vdk/.plugin-ci.yml

```diff
@@ -8,6 +8,22 @@ variables:
   DOCKER_DRIVER: overlay2
   DOCKER_TLS_CERTDIR: ""
 
+# Add all quickstart-vdk dependencies locations which trigger the dependency release when changed.
+# In this way, quickstart-vdk will always be tested before one or more dependencies are changed, and
+# this is important since quickstart-vdk has unpinned dependencies and when they are upgraded,
+# "pip install quickstart-vdk" installs their newest versions
```

Comment:
I am not super fan of adding "hard" dependencies (to the locations/paths) to other plugins / projects as it's better if they are completely independent (e.g if we can pic and move a plugin into separate repo and it would work with little changes).

 But I think we can make an exception for quickstart-vdk as it is an "aggregation" distribution for the purposes of aggregating multiple plugins in a single deliverable. 

But I wanted to note it as it should not be considered good practice. I'd add a comment

Reply:
Yes, I agree with you. I initially did not want to hard-code them, but I could not come up with a solution which is not over-complicated. I added a comment.